spec/client_spec.rb

```diff
@@ -1208,6 +1208,87 @@
     end
   end
 
+  describe '#fetch_multipart_message' do
+    describe "should raise a MissingParameterError if" do
+      it "no room_id is provided" do
+        expect {
+          @chatkit.fetch_multipart_message({})
```

Comment:
I would expect this to be passing a `message_id` but not a `room_id`. The opposite for the next test.